sync/class.jetpack-sync-actions.php

```diff
@@ -189,7 +189,9 @@ static function do_initial_sync( $new_version = null, $old_version = null ) {
 			$include_users = true;
 		}
 
-		if ( is_multisite()  ) {
+		if ( is_multisite() &&
+		     // Is Jetpack network activated
+		     in_array( 'jetpack/jetpack.php' , array_keys( get_site_option( 'active_sitewide_plugins' ) ) ) ) {
```

Comment:
let's put this under a `Jetpack::` static method, it could be useful in the future.